pkgs/tools/security/pbis/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool, perl, flex, bison, curl,
+  pam, popt, libiconv, libuuid, openssl_1_0_2, cyrus_sasl, sqlite, tdb, libxml2 }:
+
+stdenv.mkDerivation rec {
+  pname = "pbis-open";
+  version = "9.1.0";
+
+  src = fetchFromGitHub {
+    owner = "BeyondTrust";
+    repo = pname;
+    rev = version;
+    sha256 = "081jm34sf488nwz5wzs55d6rxx3sv566x6p4h1yqcjaw36174m8v";
+  };
+
+  nativeBuildInputs = [
+    autoconf automake libtool perl flex bison
```

Comment:
```suggestion
    autoreconfHook perl flex bison
```

Reply:
unfortunately it fails
```
autoreconfPhase
autoreconf: 'configure.ac' or 'configure.in' is required
builder for '/nix/store/i1yzghzcpyjbqd21wds5fdlsb65f5sw2-pbis-open-9.1.0.drv' failed with exit code 1```

Reply:
interesting, i saw autoconf, so I assumed it used configure.{ac,in}

Reply:
disregard :)